apps/desktop/src/app/sync/sources/brightcove.ts

```diff
@@ -0,0 +1,142 @@
+import {
```

Comment:
App version should be increased ? or not yet ?

Reply:
oh right! I always forget…